app/assets/javascripts/app/user_roles.js.coffee

```diff
@@ -0,0 +1 @@
+$ -> $("select[multiple].js--chosen").chosen()
```

Comment:
Seems like the filename could be a little more generic since it will technically run on every page. And I think we could even just leave it as `.js--chosen` without the `select[multiple]`.

Now's probably not the time to clean up all the very specific instantiations of chosen (e.g. `$('#product_add').chosen()` in manage_orders.coffee and add the class, but something we could think about doing.